From: Marcus Better <marcus@better.se>
Subject: do not link against libm

libm is not needed for the build, so avoid unnecessary reference.

---
 GNUmakefile |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/GNUmakefile b/GNUmakefile
index 32845c5..73bae9d 100644
--- a/GNUmakefile
+++ b/GNUmakefile
@@ -4,7 +4,6 @@ include mk/Variables.mk
 
 # add our flags + libs
 CFLAGS	+= -DVERSION='"$(VERSION)"'
-LDLIBS	+= -lm
 
 # build
 TARGETS	:= lsinput input-events input-kbd input-send input-recv lircd.conf
-- 
tg: (28d1d45..) features/exclude-libm (depends on: upstream)
